cnchi crashes when livreoffice is selected to be installed in specific languages

### Problem:
When the target language is composite, like pt_BR, the cnchi craches couse of bad package name when installing packaged. The created package should be  libreoffice-fresh-pt-BR, but it tries to use libreoffice-fresh-pt-br.

```
2016-05-23 00:15:33 [DEBUG] select_packages.py(364) add_features_packages(): Selecting package gvfs-smb for feature smb
2016-05-23 00:15:33 [DEBUG] select_packages.py(364) add_features_packages(): Selecting package python-pysmbc for feature smb
2016-05-23 00:15:33 [DEBUG] select_packages.py(382) add_features_packages(): Adding libreoffice language package (libreoffice-fresh-pt-br)
2016-05-23 00:15:33 [DEBUG] select_packages.py(400) add_features_packages(): Add firefox language package
2016-05-23 00:15:33 [DEBUG] select_packages.py(314) select_packages(): All features needed packages have been added
2016-05-23 00:15:33 [DEBUG] select_packages.py(326) select_packages(): Conflicts list: lib32-mesa-libgl, chromium, chromium-pepper-flash, ttf-droid, mesa-libgl
2016-05-23 00:15:33 [DEBUG] select_packages.py(331) select_packages(): Packages list: upower,gnome-photos,gsfonts,bash-completion,wget,bluez-utils,firefox,antergos-mirrorlist,btrfs-progs,lib32-libvdpau,baobab,pidgin,gnome-disk-utility,xorg-fonts,python-pysmbc,acpid,gnome-shell-extensions,ttf-bitstream-vera,ufw,cups,gst-plugins-bad,xorg-server,antergos-wallpapers,network-manager-applet,nfs-utils,gnome-themes-standard,gst-libav,numix-icon-theme,gnome-maps,accountsservice,gnome-keyring,flashplugin,gnome-weather,splix,antergos-keyring,light-locker,light-locker-settings,brasero,sudo,gufw,mesa-vdpau,haveged,os-prober,e2fsprogs,gnome-bluetooth,gvfs-smb,gnome-shell-extension-dash-to-dock,bijiben,unzip,pamac,ipw2200-fw,base-devel,gnome-screenshot,file-roller,xorg-server-utils,gzip,alsa-utils,networkmanager,hplip,gnome-font-viewer,xorg-xinit,libtxc_dxtn,gnome-backgrounds,gnome-sound-recorder,ipw2100-fw,ntfs-3g,ttf-google-fonts,networkmanager-pptp,libva-intel-driver,lib32-nvidia-libgl,gnome-control-center,openssh,transmission-gtk,pavucontrol,cups-pk-helper,xf86-video-fbdev,lightdm-webkit2-greeter,evince,zd1211-firmware,networkmanager-openvpn,git,gnome-terminal,cmake,tar,wpa_actiond,xscreensaver,gedit,gnome-music,gnome-tweak-tool,libdvdcss,pulseaudio,mesa,pulseaudio-alsa,net-tools,gnome-shell,gstreamer0.10-plugins,xorg-xkill,pulseaudio-bluetooth,xf86-video-vesa,gvfs-mtp,nvidia-utils,xf86-video-intel,bind-tools,gnome-system-monitor,yaourt,hicolor-icon-theme,system-config-printer,nautilus,libreoffice-fresh,libreoffice-fresh-pt-br,nvidia,cdrkit,totem,xdg-user-dirs-gtk,xf86-input-synaptics,eog,libgnomeui,gst-plugins-ugly,numix-icon-theme-square,gnome-calendar,unrar,gst-plugins-good,gnome-logs,f2fs-tools,hunspell-en,lightdm,orca,gstreamer-vaapi,nvidia-libgl,gnome-shell-extension-status-menu-buttons,grub,lib32-mesa,antergos-welcome,gnome-calculator,ttf-freefont,xfsprogs,dosfstools,bzr,exfat-utils,gnome-contacts,seahorse,whois,ffmpegthumbnailer,ttf-dejavu,dmraid,grub2-theme-antergos,wireless_tools,crda,pkgfile,antergos-gnome-defaults-list,modemmanager,ttf-liberation,reiserfsprogs,libvdpau,jfsutils,gnome-documents,gst-plugins-base,numix-frost-themes,xorg-apps,nilfs-utils,b43-fwcutter,usb_modeswitch,base
2016-05-23 00:15:33 [DEBUG] select_packages.py(109) create_package_list(): Packages selected
2016-05-23 00:15:33 [DEBUG] pac.py(113) initialize_alpm(): ALPM initialised with root dir / and db path /var/lib/pacman
2016-05-23 00:15:34 [INFO] slides.py(242) manage_events_from_cb_queue(): Criando a lista de pacotes para baixar
2016-05-23 00:15:52 [ERROR] metalink.py(124) create(): Can't find these packages: libreoffice-fresh-pt-br 
2016-05-23 00:15:52 [ERROR] download.py(145) create_metalinks_list(): Error creating metalink for package libreoffice-fresh-pt-br. Installation will stop
2016-05-23 00:15:52 [ERROR] download.py(177) create_metalinks_list(): Can't create download set. An exception of type InstallError occured. Arguments:
('Erro ao criar metalink para o pacote libreoffice-fresh-pt-br. A instalação irá parar',)
2016-05-23 00:15:52 [ERROR] process.py(129) run(): 'Não foi possivel criar a lista de pacotes de downloads (metalinks). Cheque o log de saída para mais detalhes.'
2016-05-23 00:15:52 [ERROR] process.py(133) run(): Traceback (most recent call last):
2016-05-23 00:15:52 [ERROR] process.py(133) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 94, in run
    self.create_metalinks_list()
2016-05-23 00:15:52 [ERROR] process.py(133) run():   File "/usr/share/cnchi/cnchi/installation/process.py", line 84, in create_metalinks_list
    raise misc.InstallError(txt)
2016-05-23 00:15:52 [ERROR] process.py(133) run(): misc.extra.InstallError: 'Não foi possivel criar a lista de pacotes de downloads (metalinks). Cheque o log de saída para mais detalhes.'
2016-05-23 00:15:52 [INFO] connectionpool.py(213) _new_conn(): Starting new HTTP connection (1): build.antergos.com
2016-05-23 00:15:53 [DEBUG] connectionpool.py(395) _make_request(): "GET /hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=135970&result=False HTTP/1.1" 301 None
2016-05-23 00:15:53 [DEBUG] connectionpool.py(395) _make_request(): "GET /api/hook?cnchi=2580656b5c7be34e4144a3129a95e421&install_id=135970&result=False HTTP/1.1" 200 None
```


### Steps To Reproduce:
1. Live boot and start cnchi, select Brazilian Portuguese (or other conposite language, like us_GB)
2. Select gnome
3. Select libreoffice 
4. Try to finish installation

--- a/cnchi/installation/select_packages.py
+++ b/cnchi/installation/select_packages.py
@@ -377,7 +377,7 @@ class SelectPackages(object):
                 code = self.settings.get('language_code')
 
             if code:
-                code = code.replace('_', '-').lower()
+                code = code.replace('_', '-')
                 pkg_text = "libreoffice-fresh-{0}".format(code)
                 logging.debug("Adding libreoffice language package (%s)", pkg_text)
                 self.packages.append(pkg_text)

